example/otel-collector/docker-compose.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright The OpenTelemetry Authors
+# SPDX-License-Identifier: Apache-2.0
+
+services:
+  otel-collector:
+    image: otel/opentelemetry-collector-contrib:0.91.0
```

Comment:
Should we have something that reminds us to keep those images up to date?

Reply:
Yeah, I don't see any automated way either. I also don't think it matters to upgrade every patch release.
I was more thinking of a reminder somewhere to look at it. Maybe in the release checklist? (though that adds quite a bit of toil to releasing)

Reply:
Yes, renovate can handle this.

We can handle it also using `Dockerfile` and dependabot (as it is able to bump Dockerfiles). E.g. https://github.com/signalfx/splunk-otel-go/pull/2777

Reply:
+1 for switching to renovate

Reply:
Created https://github.com/open-telemetry/opentelemetry-go/issues/5271 for switching to renovate.

Do you think we need a way to automatically verify that all examples are working after the change?